generators/pkg/writer/output/gen-task.yaml

```diff
@@ -0,0 +1,16 @@
+metadata:
```

Comment:
Can we delete this file? (I don't think it's used anywhere, it's just an output artifact)